### ISSUE TITLE
Fails if 'version' is a git ref

### ISSUE THREAD

**Issue by filipvh** [snyamathi/lerna-semver-sync#2]
`If the version is a git ref; eg: "package-name": "git+https://github.com/johndow/my-repo.git#ffg51qg2q2crg1c4g7c47" `

I get an error:
```
semver-intersect\semver-intersect.js:96
        var validRanges = validRange.split(regex.whitespace);
```

**Comment by snyamathi** [comment]
Hm these are interesting - I guess there's a few more cases where there are non numerical versions given.  Should be able to filter those out in https://github.com/snyamathi/semver-intersect

I can take a look later this week or if you'd like to submit a PR to that repo I can check it over.

Thanks for reporting though :)

**Comment by snyamathi** [comment]
Fixed in v1.3.0 https://github.com/snyamathi/lerna-semver-sync/releases/tag/v1.3.0
